python/python/bystro/ancestry/inference.py

```diff
@@ -27,6 +28,7 @@
 from bystro.utils.timer import Timer
 
 logger = logging.getLogger(__name__)
+warnings.simplefilter(action="ignore", category=FutureWarning)
```

Comment:
removes the warning about how sklearn 1.22 is using deprecated Panda's is_sparse function. Without this every time you run ancestry on the CLI you get a lot of ugly warnings that are not useful to the user.